packages/libs/api/sdk/README.md

```diff
@@ -24,8 +24,6 @@ client.nft
   .then((response) => console.log(response));
 ```
 
-Full example implementation [here](https://github.com/user/repo/blob/branch/other_file.md).
```

Comment:
Lol I hadn't noticed the link I left was broken, good catch!
that said, I think it'd still be good to point to the example app with a relative file link, wdyt?

Reply:
updated!

Reply:
🚢